src/dotnet-references/dotnet-references.csproj

```diff
@@ -24,6 +24,10 @@
     <FrameworkReference Include="Microsoft.AspNetCore.App" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == 'net6.0'">
+    <FrameworkReference Include="Microsoft.AspNetCore.App" />
+  </ItemGroup>
```

Comment:
Gonna check this as I'm surprised it's needed when it wasn't before and if it's needed for something perhaps it can be a non-aspnetcore framework. Like a Microsoft.Console.App

Reply:
Oh I see, it was there before for each, ha